pkgs/by-name/pa/paperlib/package.nix

```
@@ -0,0 +1,72 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  pkgs,
+  appimageTools,
+  undmg,
+}:
+let
+  pname = "paperlib";
+  version = "3.1.6";
+  src =
+    fetchurl
+      {
+        x86_64-darwin = {
+
+          url = "https://github.com/Future-Scholars/${pname}/releases/download/release-electron-${version}/Paperlib_${version}.dmg";
+          hash = "sha256-d9vEFx59K15PO7DJYJQ2fjiagqa8oJLtoawILDF9IKc=";
+        };
+        x86_64-linux = {
+          url = "https://github.com/Future-Scholars/${pname}/releases/download/release-electron-${version}/Paperlib_${version}.AppImage";
+          hash = "sha256-2xbn9UWlcf37n9jZdZKyyevzsag6SW9YuQH/bYCRmLQ=";
+
```

Nit formatting: (removing empty lines)

```suggestion
          url = "https://github.com/Future-Scholars/${pname}/releases/download/release-electron-${version}/Paperlib_${version}.dmg";
          hash = "sha256-d9vEFx59K15PO7DJYJQ2fjiagqa8oJLtoawILDF9IKc=";
        };
        x86_64-linux = {
          url = "https://github.com/Future-Scholars/${pname}/releases/download/release-electron-${version}/Paperlib_${version}.AppImage";
          hash = "sha256-2xbn9UWlcf37n9jZdZKyyevzsag6SW9YuQH/bYCRmLQ=";
```